contrib/testing/kind-clustermesh1.yaml

```diff
@@ -5,9 +5,11 @@ debug:
   enabled: true
 image:
   override: "localhost:5000/cilium/cilium-dev:local"
```

Comment:
it's a bit confusing to still have these referring to localhost:5000 without that being an actual registry, can we do something about that?

Reply:
I was trying to decide what to do; by default, the built images have the tag `quay.io/cilium/cilium-dev:local`.  As long as there's no chance of someone accidentally pushing that image to quay, we could keep it. Otherwise, some sort of unpushable tag would be better.